### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by danieltolmon**

### CODE REVIEW

**Comment by CRR257** [comment]
No em funciona la calculadora :(.
El diseny molt xulu :)

**Comment by danieltolmon** [comment]
> No em funciona la calculadora :(.
> El diseny molt xulu :)

File: staff/daniel-toledo/precourse/calculadora/main.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+function calculadora(num1,num2){
+
+	for(var i=0; i<arguments.length; i++){
+		if(typeof arguments[i]!='number'){
+			return 'Please insert at least 2 numbers';
+		}
+	}
+
+	function isInteger(num){
```
Comment (robert-z) [inline staff/daniel-toledo/precourse/calculadora/main.js]: Only as a little comment. I think JavaScript has a [method](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Number/isInteger) which does this.

For the rest, nice job!

File: staff/daniel-toledo/precourse/pasapalabra/pasapalabra.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
+
```
Comment (robert-z) [inline staff/daniel-toledo/precourse/pasapalabra/pasapalabra.js]: duplicate file?

File: staff/daniel-toledo/precourse/pasapalabra/bingo.js [hunk 3]
```diff
@@ -0,0 +1,268 @@
+//BINGO - PRO
```
Comment (robert-z) [inline staff/daniel-toledo/precourse/pasapalabra/bingo.js]: duplicate file?

Comment (danieltolmon) [reply staff/daniel-toledo/precourse/pasapalabra/bingo.js]: In Pasapalabra project I acttualy did a CASINO SKYLAB, where you can choose to play either bingo or pasapalabra. That's why you can find a duplicate file

**Pull request merged**